.github/workflows/scripts/release-crates-dry-run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -ex
+
+# Check if necessary tools are installed
+if ! command -v git &> /dev/null || ! command -v cargo &> /dev/null; then
+    echo "ERROR: Required tools (git, cargo) are not installed."
+    exit 1
+fi
+
+git config --global user.email "release-tests-no-reply@zfnd.org"
+git config --global user.name "Automated Release Test"
+
+# Ensure cargo-release is installed
+if ! cargo release --version &> /dev/null; then
+    echo "ERROR: cargo release must be installed."
+    exit 1
+fi
+
+# Release process
+# Ensure to have an extra `--no-confirm` argument for non-interactive testing.
+cargo release version --verbose --execute --no-confirm --allow-branch '*' --workspace --exclude zebrad beta
+cargo release version --verbose --execute --no-confirm --allow-branch '*' --package zebrad patch
+cargo release replace --verbose --execute --no-confirm --allow-branch '*' --package zebrad
+cargo release commit --verbose --execute --no-confirm --allow-branch '*'
```

Comment:
@arya2 do your release script changes also need to be made here?

(Or maybe we can just assume we'll make the scanner into a beta version soon, because we've almost finished the MVP.)

Reply:
Let's just assume the scanner will become beta soon.